src/styles/index.css

```diff
@@ -0,0 +1,238 @@
+  margin-top: 4px;
+  padding: 0 16px;
+  width: 100%;
+  justify-content: center;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+.card__reviews {
+  display: flex;
+  margin: 16px;
+  margin-bottom: 0;
+  width: 166px;
+  justify-content: space-between;
+}
+.card__reviews .reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 16px;
+  color: #060b35;
+  text-align: right;
+}
+.card__price {
+  display: flex;
+  width: 166px;
+  line-height: 18px;
+  justify-content: space-between;
+  margin: 24px 16px 0;
+}
+.card__price .price {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+}
+.card__price .amount {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+}
+.card .button {
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  width: 166px;
+  height: 40px;
+  margin: 16px;
+  border-color: #00acdc;
+  border-radius: 5px;
+  background: #00acdc;
+  transition: background-color 300ms, border 300ms;
+}
+.card .button__buy {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+  transition: color 300ms;
+}
+.card .button__buy:hover {
+  color: #00acdc;
+}
+.card .button:hover {
+  background-color: white;
+  border: 1px solid #00acdc;
+}
+.card:hover {
+  transform: scale(1.2);
+}
+.card:hover .card__title {
+  color: #34568b;
+}
+
+.header {
+  width: 100vw;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 50px;
+  width: 100%;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.0509803922);
+}
+.header__container .logo__img {
+  display: flex;
+  padding-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+.header__container .nav {
+  display: flex;
+}
+.header__container .nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  align-items: center;
+}
+.header__container .nav__link.is-active {
+  margin-left: 0;
+  text-align: center;
+  display: flex;
+  line-height: 60px;
+  color: #00acdc;
+  text-decoration: none;
+  transition: color 300ms;
+}
+.header__container .nav__link {
+  margin-left: 20px;
+  text-align: center;
+  display: flex;
+  line-height: 60px;
+  color: #060b35;
+  text-decoration: none;
+  transition: color 300ms;
+}
+.header__container .nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+  text-decoration: none;
+}
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+}
+
+.stars {
+  display: flex;
+  border-radius: 0.5px;
```

Comment:
The border-radius value of `0.5px` is quite small and might not be noticeable. Consider using a larger value if you want the rounded corners to be more visible.